lib/yam/agama/agama_base.pm

```diff
@@ -0,0 +1,20 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: base class for Agama tests
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package yam::agama::agama_base;
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
```

Comment:
It's better to just include function used in the lib, for this only 'select_console'.

Reply:
I assume agama_base will become bigger and will probably use more things from testapi. But for now you're right.